_sass/blocks/jekyll-layouts/_content.scss

```diff
@@ -6,14 +6,15 @@
 
     p {
       @include heading('para-xl');
+      margin-top: 2.25rem;
```

Comment:
Could this be something in variables? [`$standard-padding-large`](https://github.com/18F/doi-extractives-data/blob/bb6b321ed9d83493867696e905ae2461c0c2d115/_sass/_variables.scss#L112)? 1/875rem